mailet/standard/src/main/java/org/apache/james/transport/mailets/ClamAVScan.java

```diff
@@ -694,6 +697,10 @@ public void service(Mail mail) throws MessagingException {
 
     }
 
+    private Attribute makeAttribute(boolean value) {
```

Comment:
these makeAttribute methods are ugly, do we really need them?

Reply:
we do not need them, but I find it more easy to read than specifying to complete `Attribute` definition.